pkgs/applications/misc/bicon/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+  , fetchFromGitHub
+  , autoreconfHook
+  , pkg-config
+  , perl
+  , fribidi
+  , kbd
+  , xkbutils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bicon";
+  version = "unstable-2018-09-10";
+
+  src = fetchFromGitHub {
+    owner = "behdad";
+    repo = pname;
+    rev = "38725c062a83ab19c4e4b4bc20eb9535561aa76c";
+    sha256 = "0hdslrci8pq300f3rrjsvl5psfrxdwyxf9g2m5g789sr049dksnq";
+  };
+
+  buildInputs = [ fribidi kbd xkbutils perl ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+
+  preConfigure = ''
+    patchShebangs ./autogen.sh
+    patchShebangs ./configure
+    patchShebangs ./font/bicon-bdf2psf.pl
+  '';
+
+  configureFlags = [
+    "--enable-fribidi"
+    "--with-gtk=2.0,3.0"
+  ];
```

Comment:
I can't see these options in the README https://github.com/behdad/bicon/blob/38725c062a83ab19c4e4b4bc20eb9535561aa76c/README
or configure.ac https://github.com/behdad/bicon/blob/38725c062a83ab19c4e4b4bc20eb9535561aa76c/configure.ac
If you look at the build log, it says:
```
configure: WARNING: unrecognized options: --enable-fribidi, --with-gtk
```
So, I guess, we don't need them.